src/components/Markdown/MetaData.tsx

```diff
@@ -8,12 +8,13 @@ interface MetaDataProps {
 }
 
 const MetaData = ({ title, description, date }: MetaDataProps) => {
+	const formattedTitle = `Kastel -  ${title}`;
```

Comment:
```suggestion
	const formattedTitle = `Kastel -  ${title}`;

```

should add a new line before the return besides that all good :p